src/formats/prometheus_text_format.erl

```diff
@@ -105,10 +109,12 @@ emit_series(Fd, Name, Labels, Value) ->
 escape_metric_help(Help) ->
   sub(sub(Help, "\\", "\\\\\\\\"), "\n", "\\\\n").
 
+%% FIXME: -spec escape_label_value(term()) -> string().
 escape_label_value(LValue) when is_list(LValue)->
   sub(sub(sub(LValue, "\\", "\\\\\\\\"), "\n", "\\\\n"), "\"", "\\\\\"");
 escape_label_value(LValue) when is_binary(LValue) ->
   escape_label_value(binary_to_list(LValue));
+%% FIXME: dialyzer says this "can never match"
```

Comment:
ok I'm using `infinity` atom as mandatory last bucket exploiting the fact the all numbers all less than other types. I'll replace `infinity` with `<<"infinity">> then.